src/components/test-slot/additional-candidate-details/additional-candidate-details.scss

```diff
@@ -0,0 +1,27 @@
+.additional-candidate-details {
+  margin-left: 10px;
+
+  &__list {
+    margin: 0;
+  }
+
+  &__key {
+    display: inline-block;
+
+    &:nth-of-type(n+2) {
+      margin-left: 5px;
+
+      &::before {
+        content: '';
+        border-left: 1px solid black;
```

Comment:
we should probably use gds-black here

Reply:
Checked elsewhere and couldn't see a variable in use for the surrounding text so used the exact same. I can switch it but will have to update others as well to ensure consistency.